api/src/__init__.py

```diff
@@ -0,0 +1,121 @@
+import requests
+from typing import List
+
+# ---------- GitHub API Classes ----------
+
+class GitHubIssue:
+    def __init__(self, issue_id, title, body, url, state):
+        self.id = issue_id
+        self.title = title
+        self.body = body
+        self.url = url
+        self.state = state
+
+class GitHubAPI:
+    def __init__(self, token: str):
+        self.token = token
+        self.headers = {
+            "Authorization": f"token {self.token}",
+            "Accept": "application/vnd.github+json"
+        }
+
+    def fetch_issues(self, owner: str, repo: str) -> List[GitHubIssue]:
+        url = f"https://api.github.com/repos/{owner}/{repo}/issues"
+        response = requests.get(url, headers=self.headers)
+        response.raise_for_status()
+        return [
+            GitHubIssue(
+                issue_id=item["number"],
+                title=item["title"],
+                body=item.get("body", ""),
+                url=item["html_url"],
+                state=item["state"]
+            )
+            for item in response.json()
+        ]
+
+    def close_issue(self, owner: str, repo: str, issue_id: int) -> bool:
+        url = f"https://api.github.com/repos/{owner}/{repo}/issues/{issue_id}"
+        data = {"state": "closed"}
+        response = requests.patch(url, headers=self.headers, json=data)
+        return response.status_code == 200
+
+    def comment_on_issue(self, owner: str, repo: str, issue_id: int, comment: str) -> bool:
+        url = f"https://api.github.com/repos/{owner}/{repo}/issues/{issue_id}/comments"
+        response = requests.post(url, headers=self.headers, json={"body": comment})
+        return response.status_code == 201
+
+
+# ---------- Trello API Classes ----------
+
+class TrelloCard:
+    def __init__(self, card_id, name, desc):
+        self.id = card_id
+        self.name = name
+        self.desc = desc
+
+class TrelloAPI:
+    def __init__(self, key: str, token: str):
+        self.key = key
+        self.token = token
+
+    def create_card(self, list_id: str, card: TrelloCard) -> bool:
+        url = "https://api.trello.com/1/cards"
```

Comment:
Currently you are tying your issue tracker client with trello. We don't want any dependencies for our interface.
It should be defined such that implementation for any interface can be dependency injected.